test/src/template/array_selector_test.dart

```diff
@@ -63,8 +62,8 @@ class EmployeeList extends PolymerElement {
   }
 
   @reflectable
-  void toggleSelection(MouseEvent e, [_]) {
-    var item = new DomRepeatModel.fromEvent(e).item;
+  void toggleSelection(e, [_]) {
+    var item = new DomRepeatModel.fromEvent(e)['item'];
```

Comment:
Ya, this is now a `CustomEventWrapper` because apparently it has a `detail` field.